src/test/fly/fly.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "../../schemas/json/fly.json",
```

Comment:
Could this be removed? For consistency with the other test files.

Reply:
removed